### PR TITLE
regenerate usage-api

### PR DESCRIPTION
**Pull request by AlexTugarev**
## Description

Required changes to `enum System` blocked `usage-api` generation. This PR follows applies the changes resulting from the introduction of the enum. 

## Release Notes

```release-note
NONE
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

File: components/server/src/workspace/workspace-starter.ts [hunk 1]
```diff
@@ -1435,17 +1436,23 @@ export class WorkspaceStarter {
...
         if (lastValidWorkspaceInstanceId != "") {
-            volumeSnapshotId = lastValidWorkspaceInstanceId
-        } else if ((SnapshotContext.is(workspace.context) || WithPrebuild.is(workspace.context)) && !!workspace.context.snapshotBucketId) {
-            volumeSnapshotId = workspace.context.snapshotBucketId
+            volumeSnapshotId = lastValidWorkspaceInstanceId;
+        } else if (
+            (SnapshotContext.is(workspace.context) || WithPrebuild.is(workspace.context)) &&
+            !!workspace.context.snapshotBucketId
+        ) {
+            volumeSnapshotId = workspace.context.snapshotBucketId;
```
Comment (geropl) [inline components/server/src/workspace/workspace-starter.ts:1448]: This changes behavior, and looks like a rebase error. @AlexTugarev can you confirm?

**Review by geropl — CHANGES_REQUESTED** [review]
https://github.com/gitpod-io/gitpod/pull/12294/files#r952273653

File: components/usage/pkg/db/billed_session.go [hunk 2]
```diff
@@ -30,10 +33,19 @@ func (d *BilledSession) TableName() string {
...
-	billedSession := BilledSession{InstanceID: instanceID, From: NewVarcharTime(instanceCreationTime), System: system}
+	var sys string
+	switch system {
+	case v1.System_SYSTEM_CHARGEBEE:
+		sys = "chargebee"
+	case v1.System_SYSTEM_STRIPE:
+		sys = "stripe"
+	default:
+		sys = "unknown"
+	}
```
Comment (easyCZ) [inline components/usage/pkg/db/billed_session.go:47]:
```suggestion
	var sys = "unknown"
	switch system {
	case v1.System_SYSTEM_CHARGEBEE:
		sys = "chargebee"
	case v1.System_SYSTEM_STRIPE:
		sys = "stripe"
	}
```

File: components/server/src/workspace/workspace-starter.ts (hunk 1 above)
Comment (AlexTugarev) [reply components/server/src/workspace/workspace-starter.ts:1448]: @geropl, could you clarify, please. 
Cannot see any significant changes. I touched the `System` param in that file, and it got formatted on save. Formatter changes look ok to me.

Comment (geropl) [reply components/server/src/workspace/workspace-starter.ts:1448]: @AlexTugarev Sorry, thought I'd seen an additional `else` branch here. :see_no_evil: Definitely gone now. :stuck_out_tongue: 
Sorry for the noise!

**Review by geropl — APPROVED** [review]
:+1:

File: components/server/src/workspace/workspace-starter.ts (hunk 1 above)
Comment (AlexTugarev) [reply components/server/src/workspace/workspace-starter.ts:1448]: No problemo 👍🏻

**Comment by aledbf** [comment]
@AlexTugarev please rebase. The fix for the build is merged now https://github.com/gitpod-io/gitpod/pull/12299

**Pull request merged**